### PR TITLE
Make unicam_isp_capture possible to vflip and hflip

### PR DESCRIPTION
**Pull request by amutake**
## Check list

- [x] I wrote [CHANGELOG.md](./CHANGELOG.md) if the pull request adds/modifies features.

## Summary

We have made unicam_isp_capture possible to vflip and hflip.

We confirmed with imx219 and ov5647 sensors.

### CODE REVIEW

**Review by riku179 — APPROVED** [review]
LGTM 👍

File: actfw_core/v4l2/video.py [hunk 1]
```diff
@@ -377,6 +377,8 @@ class MEDIA_BUS_FMT(enum.IntEnum):
     SBGGR8_1X8 = 0x3001
     SBGGR10_1X10 = 0x3007
     SGBRG10_1X10 = 0x300E
+    SGRBG10_1X10 = 0x300A
+    SRGGB10_1X10 = 0x300F
```
Comment (cutsea110) [inline actfw_core/v4l2/video.py:381]: OK!

**Review by cutsea110 — APPROVED** [review]
ありがとうございます！LGTM

**Comment by amutake** [comment]
すみませんコンフリクト解消したら approve 取れてしまったのでもう一度お願いします :pray:

**Review by riku179 — APPROVED** [review]

**Pull request merged**